person.rb

```diff
@@ -2,18 +2,19 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1...1000)
-    @name = name
     @age = age
+    @name = name
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
   end
 
-  def validate_name
-    @name = corrector.correct_name(@name)
+  def validate_name(name)
+    @name = @corrector.correct_name(name)
   end
 
```

Comment:
- [ ] A method for adding rentals should be present.
- [ ] The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
```
def add_rental(book, date)
    Rental.new(date, book, self)
end
```

Reply:
Thank you, I will add it immediately.